packages/webdriverio/README.md

```diff
@@ -33,26 +33,29 @@ This module uses a chainable API to assist in injecting, configuring, and analyz
 Here is an example of a script that will drive WebdriverIO to a page, perform an analysis, and then log results to the console.
 
 ```js
-const AxeBuilder = require('@axe-core/webdriverio').default;
+const { AxeBuilder } = require('@axe-core/webdriverio');
 const { remote } = require('webdriverio');
 
 (async () => {
   const client = await remote({
     logLevel: 'error',
     capabilities: {
-      browserName: 'firefox'
```

Comment:
Firefox requires a binary that isn't installed by default on macs, so switched to chrome